src/uu/dircolors/src/dircolors.rs

```diff
@@ -262,11 +282,12 @@ where
     T: IntoIterator,
     T::Item: Borrow<str>,
 {
-    // 1440 > $(dircolors | wc -m)
-    let mut result = String::with_capacity(1440);
+    // 1790 > $(dircolors | wc -m)
+    let mut result = String::with_capacity(1790);
```

Comment:
Where does this change come from? Does is have anything to do with the change you made or did you just find a different value?

Reply:
This is a change related to #3559 I didn't notice back then. The value is rounded up from what `dircolors | wc -m` returns (1784).

Reply:
Alright thanks!